src/services/users.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+const { v4: uuidv4 } = require('uuid');
+
+let users = [];
+// let count = 0;
+
+function getAll() {
+  return users;
+}
+
+function getById(userId) {
+  const foundUser = users.find(user => user.id === userId);
+
+  return foundUser || null;
+}
+
```

Comment:
Missing validation for the 'name' parameter. It should check if the provided name is not empty or undefined before creating a new user.

Reply:
Seeems like I've deleted it with console statements)) Done